Floofbot/Modules/Fun.cs

```diff
@@ -32,19 +32,27 @@ public async Task AskEightBall([Summary("question")][Remainder] string question)
         public async Task XKCD([Summary("Comic ID")] string comicId = "")
         {
             int parsedComicId;
-            if (!int.TryParse(comicId, out parsedComicId) || parsedComicId <= 0)
+            if ((!int.TryParse(comicId, out parsedComicId) || parsedComicId <= 0) && !comicId.ToLower().Equals(""))
```

Comment:
For readability I'd suggest using `String.IsNullOrEmpty` instead

Reply:
Ah, good point, will push that change once I get back to the laptop.